src/main/java/seedu/duke/commands/member/AssignMemberToProjectCommand.java

```diff
@@ -1,55 +1,60 @@
+//@@author thatseant
+
 package seedu.duke.commands.member;
 
-import java.lang.reflect.Array;
 import java.util.HashMap;
 import seedu.duke.DukeExceptions;
 import seedu.duke.commands.Command;
 import seedu.duke.member.TeamMember;
 import seedu.duke.project.Project;
+import seedu.duke.ui.Ui;
 
 import static seedu.duke.Parser.getHashValue;
+import static seedu.duke.Util.INDEX_NON_INTEGER;
+import static seedu.duke.Util.INVALID_PROJECT_ID;
+import static seedu.duke.Util.INVALID_TEAM_MEMBER_ID;
+import static seedu.duke.Util.MEMBER_INDEX_KEY;
+import static seedu.duke.Util.PROJECT_INDEX_KEY;
+import static seedu.duke.Util.USER_JAVA_INDEX_DIFF;
+
 import java.util.ArrayList;
 
 public class AssignMemberToProjectCommand extends Command {
     private int memberIndex;
     private int projectIndex;
-    private boolean isHomeView;
     private HashMap<String, String> paramsList;
 
-    public AssignMemberToProjectCommand(HashMap<String,String> paramsList, boolean isHomeView)
+    public AssignMemberToProjectCommand(HashMap<String,String> paramsList, int projectIndex)
             throws DukeExceptions {
         this.paramsList = paramsList;
-        this.isHomeView = isHomeView;
+        assert projectIndex == -1 : projectIndex;
```

Comment:
Good use of assertions. 
![image](https://user-images.githubusercontent.com/60358173/98460216-2bb98680-21dd-11eb-844f-dc95e31404ff.png)

Maybe the error could be clearer?? as seen above

Reply:
Thanks Samuel! I'll change it